ufc/bandit-flags-v1.json

```diff
@@ -201,6 +201,31 @@
                 }
             ],
             "totalShards": 10000
+        },
+        "car_bandit_flag": {
```

Comment:
A new bandit so we can safely play with the parameters without messing up existing test cases. For simplicity, we assign 100% of the traffic to this bandit